connectors/agentic-ai/src/main/java/io/camunda/connector/agenticai/a2a/client/A2AClientFunction.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.agenticai.a2a.client;
+
+import io.camunda.connector.agenticai.a2a.client.model.A2AClientRequest;
+import io.camunda.connector.agenticai.a2a.client.model.result.A2AClientResult;
+import io.camunda.connector.api.annotation.OutboundConnector;
+import io.camunda.connector.api.outbound.OutboundConnectorContext;
+import io.camunda.connector.api.outbound.OutboundConnectorFunction;
+import io.camunda.connector.generator.java.annotation.ElementTemplate;
+
+@OutboundConnector(
+    name = "A2A Client",
+    inputVariables = {"data"},
+    type = "io.camunda.agenticai:a2aclient:0")
+@ElementTemplate(
+    id = "io.camunda.connectors.agenticai.a2a.client.v0",
+    name = "A2A Client (experimental)",
+    description =
+        "Agent-to-Agent (A2A) client, enabling discovering remote agents' Agent Cards as well as invoking remove agents. Compatible with 8.8.0-alpha8 or later.",
+    engineVersion = "^8.8",
```

Comment:
I think we can assume this will be 8.9 only, right?

```suggestion
    engineVersion = "^8.9",
```

Reply:
Yes, it should be `8.9` only :+1: